hack/update-generated-publishing-rules.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+os::build::setup_env
+
+# NOTE: You need to have installed these two packages locally:
+# go get -u github.com/Masterminds/glide
+# go get -u github.com/kubernetes/publishing-bot
+#
+os::util::ensure::built_binary_exists 'genrules' 'tools/genrules'
+
+genrules -mappings="master:release-1.9.1" > ./hack/publishing-rules.yaml
```

Comment:
Maybe comment: this file does only play a role in the master branch. Do not update this list on release branches.

Reply:
you have to update this list if you want to sync non-master branches... i would start with master now as 1.8.x branch in origin seems to be broken somehow (bad rebase)...

Reply:
I mean, when we cut release-1.10.0 branch this will have to change to:

```
-mappings="master:release-1.10.0,release-1.9.1:release-1.9.1"
```